functions-methods.js

```diff
@@ -33,4 +64,26 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+function checkEmailValidity(email) {
+    const containsMonkeyTail = email.includes("@");
+    const containsComma = email.includes(",");
+
+    const indexOfLastDot = email.lastIndexOf(".");
+    const containsNoDotAtEnd = indexOfLastDot !== email.length - 1;
+
+    if (containsMonkeyTail && !containsComma && containsNoDotAtEnd) {
+        return true;
+    } else {
+        return false;
+    }
```

Comment:
Een leuke truc om te weten: we kunnen de if-statement en return nog korter maker! We willen namelijk alleen een boolean terug ontvangen. We kunnen dit korter maken door:
` return !!(containsMonkeyTail && !containsComma && containsNoDotAtEnd);` te schrijven. De dubbele uitroeptekens zijn namelijk ook een conditionele check, een soort van "mini" if-statement! :)